### ISSUE TITLE
Fix color contrast issues

### ISSUE THREAD

**Issue by davgross** [davgross/homeless-in-slo-resource-guide#20]
Our accessibility audit found a few areas where there is insufficient contrast between the text color and the background color:

| Foreground | Background | Use Case | Min Ratio | Status | Consider instead? |
|---|---|---|---|---|---|
| `#3877ff` | `#ffffff` | Link text | 4.5:1 | 3.99:1 (failure) | `#2469FF` |
| `#ffffff` | `#3877ff` | Nav buttons | 4.5:1 | 3.99:1 (failure) | `#2469FF` |
| `#e75e13` | `#ffffff` | External links | 4.5:1 | 3.49:1 (failure) | `#c65010` |
